public/locales/es/about.json

```diff
@@ -33,6 +33,23 @@
         "part2": "Los sensores PurpleAir nos caen bien porque son económicos y relativamente precisos considerando su precio. Dicho esto, no son tan confiables como algunos sensores más caros, lo que significa que a veces los valores que producen los sensores son insuficientes para calcular el AQI. Esto podría ser muy temporal (una cuestión de minutos u horas) o podría durar más tiempo. Si un sensor ha estado inactivo durante mucho tiempo, probablemente significa que el sensor ya no está conectado al Internet o que tiene un problema de hardware más grave.",
         "part3": "Si un sensor en tu área ha estado inactivo durante mucho tiempo, por favor contáctanos y nos comunicaremos con el anfitrión del sensor para tratar de resolver el problema."
     },
+    "app":{
+        "heading": "Descargue este sitio web como una aplicación",
+        "body": "Si tiene un smartphone Android o un iPhone, puede descargar este sitio web como una aplicación en su pantalla de inicio siguiendo estas instrucciones.",
```

Comment:
Maybe use instalar instead of descargar

Reply:
Also definitely have Hillary check these, I tried checking on my phone to see what it said but now my phone language is all confused lol